include/camp/list/list.hpp

```diff
@@ -1,13 +1,16 @@
 #ifndef CAMP_LIST_LIST_HPP
 #define CAMP_LIST_LIST_HPP
 
+#include "camp/number/number.hpp"
```

Comment:
@trws we've been seeing some weird warnings in appveyor related to the contents of the 'number.hpp' header file. Will you please take a look and let us (me and @davidbeckingsale) if this is a real issue or some strangeness with MSVC)?  Thanks.

Reply:
I'll make a note to look at that, thanks for the heads-up.